config/routes.rb

```diff
@@ -23,9 +23,12 @@
   get '/v0/sign_in/logingov_logout_proxy', to: 'v0/sign_in#logingov_logout_proxy'
   get '/v0/sign_in/revoke_all_sessions', to: 'v0/sign_in#revoke_all_sessions'
 
-  get '/v0/sign_in/client_config', to: 'v0/sign_in#read_client_config'
   get '/sign_in/openid_connect/certs' => 'sign_in/openid_connect_certificates#index'
 
+  namespace :sign_in do
```

Comment:
I think we should disallow this from being called in production for now (or disallow the create/destroy/update ones if nothing else). Just want to make sure we're being very careful. When we do enable these in production, they will have to have an additional layer of security as well, 'like service account authenticate but make sure the user making the action is within this set of emails'